backend/src/main/java/com/redhat/cloud/notifications/routers/EndpointResource.java

```diff
@@ -227,11 +257,13 @@ public EndpointPage getEndpoints(
         @QueryParam("name")     String name
     ) {
         if (this.backendConfig.isKesselBackendEnabled()) {
-            // Check that the principal has the proper permission to list the
-            // endpoints.
-            this.kesselAuthorization.hasPermissionOnResource(sec, WorkspacePermission.INTEGRATIONS_READ, ResourceType.WORKSPACE, "TODO get workspace from request");
+            // Fetch the set of integration IDs the user is authorized to view.
+            final Set<UUID> authorizedIds = this.kesselAuthorization.lookupAuthorizedIntegrations(sec, IntegrationPermission.VIEW);
+            if (authorizedIds.isEmpty()) {
+                return new EndpointPage(new ArrayList<>(), new HashMap<>(), new Meta(0L));
```

Comment:
Could you please add a log to track this case? it should be rare, but could be tricky to debug as well.

Reply:
I've chosen `INFO` as the log level for this log message, because a `WARNING` would bug us too much in the alert channels, I think, and a `DEBUG` statement might not be enough now that we are on the initial stages of the integration?

I am doubtful, though :thinking: 

The Kessel client has lots of `debug` and `trace` statements, so maybe we should set it to `DEBUG`?

Reply:
I think `INFO` is fine for this.